scimono-client/src/test/java/com/sap/scimono/client/SCIMResourceRequestInvocationTest.java

```diff
@@ -213,6 +211,23 @@ private DynamicTest readMultipleUsersInvocationDefaultPaging() {
     });
   }
 
+  private DynamicTest readMultipleUsersInvocationWithRequestedAttributes() {
+    return dynamicTest("Read multiple Users invocation with requested and excluded attributes", () -> {
+      Collection<Executable> capturedAssertions = new ArrayList<>();
+      String expectedFilter = DEFAULT_URL + "/Users?attributes=displayName&excludedAttributes=groups";
+
+      assertThrows(ProcessingException.class,
```

Comment:
Minor: I would rather extract the UserRequest and the RequestDetails initialization before the assertThrows clause. This way it will be more straightforward to notice that we expect the readMultipleUsers to throw the exception. 
Although I see that all the tests in this class are done this way, so it's really up to you.

Reply:
Done